lib/msal-browser/src/interaction_handler/PopupHandler.ts

```diff
@@ -41,7 +42,7 @@ export class PopupHandler extends InteractionHandler {
             this.browserStorage.setTemporaryCache(TemporaryCacheKeys.INTERACTION_STATUS_KEY, BrowserConstants.INTERACTION_IN_PROGRESS_VALUE, true);
             this.browserRequestLogger.infoPii(`Navigate to: ${requestUrl}`);
             // Open the popup window to requestUrl.
-            return this.popupUtils.openPopup(requestUrl, params.popupName, params.popup);
+            return this.popupUtils.openPopup(requestUrl, params.popupName, params.popupWindowAttributes, params.popup);
```

Comment:
can you just pass `params` here instead of each property separately?

Reply:
Good idea. Done.